go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/go-openapi/jsonreference v0.19.3
 	github.com/go-openapi/swag v0.19.5
 	github.com/golang/protobuf v1.5.2
-	github.com/google/gnostic v0.6.6
+	github.com/google/gnostic v0.5.7-v3refs
```

Comment:
this isn't a semantic version, which means `go mod tidy` wasn't run on this, right?

Reply:
I did run `go mod tidy` on this, which removed the jsonschema dependencies. The version name is valid per semver https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions (pre-release).

Reply:
huh... I didn't realize go preserved that... that's fun